pydicer/convert/rtstruct.py

```diff
@@ -25,10 +25,11 @@ def convert_rtstruct(
     Args:
         dcm_img_list (list): List of DICOM paths (as str) to use as the reference image series.
         dcm_rt_file (str|pathlib.Path): Path to the DICOM RTSTRUCT file
-        prefix (str, optional): The prefix to give the output files. Defaults to "Struct_".
+        prefix (str, optional): The prefix to give the output files.
+        Defaults to "Struct" + underscore.
         output_dir (str|pathlib.Path, optional): Path to the output directory. Defaults to ".".
```

Comment:
Does this still render properly? One of the reasons we kept these indented was so that the text would be appended to the text in the line above. Like this it might render "Defaults" as an attribute for the function? Not sure.

Reply:
Yeah fixed that issue now, just installed an extension to suppress the sphinx error. You can see the docs here now https://australiancancerdatanetwork.github.io/pydicer/html/index.html

Reply:
Awesome thanks @dalmouiee